stdlib/ballerina-http/src/main/java/org/ballerinalang/net/ws/Constants.java

```diff
@@ -67,4 +67,7 @@
     public static final String NATIVE_DATA_UPGRADE_HEADERS = "NATIVE_DATA_UPGRADE_HEADERS";
     public static final String NATIVE_DATA_PARENT_CONNECTION_ID = "NATIVE_DATA_PARENT_CONNECTION_ID";
     public static final String NATIVE_DATA_PING_TIME_VALIDATOR = "NATIVE_DATA_PING_TIME_VALIDATOR";
+
```

Comment:
Rename this Constant class. We cannot have classes called Constants. 

There are few such classes in the source code. We need to refactor those as well.

Reply:
Since there are several Constants classes and even renaming one would cause lots of code changes I have created an issue to address this: https://github.com/ballerina-lang/ballerina/issues/4558 . 
I am not renaming this class in this PR because it will change files that are not related to this PR.

Reply:
Great.